### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by rahulkumaran**
This is wrt #272 
Updated year in license and is up to date now.

### CODE REVIEW

**Comment by rahulkumaran** [comment]
Just wanted to know when this would be merged @productdana

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 SendGrid
+Copyright (c) 2017-2018 SendGrid
```
Comment (Ditofry) [inline LICENSE]: @rahulkumaran  Thanks for the contribution!  I apologize for the delay.  We will merge this very soon.  May I ask, though, that you update to just the year per the MIT standard: https://opensource.org/licenses/MIT

Thanks, and happy hacking!

Comment (rahulkumaran) [reply LICENSE]: Sure! I'll update the year to only 2018!

**Comment by thinkingserious** [comment]
Hello @rahulkumaran,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 4 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Pull request closed without merging**